### PR TITLE
[MBL-17636][Student][Teacher] Calendar create title a11y

### PR DESCRIPTION
**Pull request by hermannakos**
Test plan: See ticket, design in comments.

refs: MBL-17636
affects: Student, Teacher
release note: none

## Screenshots

<table>
<tr><th>Before</th><th>After</th></tr>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
</table>

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: libs/pandautils/src/main/java/com/instructure/pandautils/features/calendarevent/createupdate/composables/CreateUpdateEventScreen.kt [hunk 1]
```diff
@@ -603,3 +633,19 @@ private fun CreateUpdateEventPreview() {
         actionHandler = {}
     )
 }
+
+@Preview
+@Composable
+private fun TitleInputPreview() {
+    TitleInput("really really really really really really really really really long",
```
Comment (kdeakinstructure) [inline libs/pandautils/src/main/java/com/instructure/pandautils/features/calendarevent/createupdate/composables/CreateUpdateEventScreen.kt:640]: 😃

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Review by tamaskozmer — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**